admin/category_product_listing.php

```diff
@@ -679,30 +683,30 @@
                   <td class="hidden-sm hidden-xs"><!-- no model for categories --></td>
                   <td class="hidden-sm hidden-xs"><!-- no price for categories --></td>
                 <?php } ?>
-                <?php if ($search_result || SHOW_COUNTS_ADMIN == 'true') { ?>
+                <?php if ($search_result || SHOW_COUNTS_ADMIN === 'true') { ?>
                   <td class="text-right hidden-sm hidden-xs">
                     <?php
-                    if (SHOW_COUNTS_ADMIN == 'true') {
+                    if (SHOW_COUNTS_ADMIN === 'true') {
                       // show counts
                       $total_products = zen_get_products_to_categories($category['categories_id'], true);
-                      $total_products_on = zen_get_products_to_categories($category['categories_id'], false);
+                      $total_products_on = zen_get_products_to_categories($category['categories_id']);
                       echo $total_products_on . TEXT_PRODUCTS_STATUS_ON_OF . $total_products . TEXT_PRODUCTS_STATUS_ACTIVE;
                     }
                     ?>
                   </td>
                 <?php } ?>
                 <td class="text-right dataTableButtonCell">
-                  <?php if (SHOW_CATEGORY_PRODUCTS_LINKED_STATUS == 'true' && zen_get_products_to_categories($category['categories_id'], true, 'products_active') == 'true') { ?>
+                  <?php if (SHOW_CATEGORY_PRODUCTS_LINKED_STATUS === 'true' && zen_get_products_to_categories($category['categories_id'], true, 'products_active') == 'true') { ?>
```

Comment:
Why not `===` for the check on zen_get_products_to_categories?

Reply:
That function needs attention in general...

But, currently returns string '' or string 'true' or an integer count (function description is incorrect).
In this case, using 'products_active', it returns 'true'.